panel/layout.py

```diff
@@ -254,21 +267,27 @@ class Tabs(Panel):
     _linked_props = ['active']
 
     def __init__(self, *items, **params):
+        objects, self._names = self._to_objects_and_names(items)
+        super(Tabs, self).__init__(*objects, **params)
+        self.param.watch(self._update_names, 'objects')
+
+    def _to_object_and_name(self, item):
```

Comment:
This method and the one below are helper functions and don't depend on self, so they could be moved out of the class, but I thought it was clearer to have them as class methods.

Reply:
Seems fine to me, could be classmethod but I don't really mind.